rig/machine_control/__init__.py

```diff
@@ -1,2 +1,36 @@
+"""A collection of utilities for controlling the operation of a SpiNNaker
+machine.
+
+SpiNNaker software can be controlled through the use of the SpiNNaker Control
+Protocol (SCP) and hardware can be controlled through the use of the BMP
+system.  A :py:class:`~MachineController` wraps up these interfaces and
```

Comment:
"SpiNNaker software and board management processors (BMPs) can be controlled through the use of the SpiNNaker Control Protocol (SCP)."